front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/formatTrainSchedulePayload.ts

```diff
@@ -25,7 +25,7 @@ export default function formatTrainSchedulePayload(
     initial_speed: initialSpeed,
     labels,
     // TODO TS2 : handle margins
-    // margins: validConfig.margins,
+    margins: validConfig.margins,
```

Comment:
You can extract margins and schedule from validConfig l11, like the other args :)

Reply:
Done